### ISSUE TITLE
Version 3.3.2 is missing AAR files in Maven, blocking android builds

### ISSUE THREAD

**Issue by fromkeith** [bugfender/BugfenderSDK-android-sample#18]
The file `android-3.3.2.aar` is missing from maven

listing the available files: https://repo.maven.apache.org/maven2/com/bugfender/sdk/android/3.3.2/

```
android-3.3.2-javadoc.jar                         2024-11-07 13:55    135342      
android-3.3.2-javadoc.jar.asc                     2024-11-07 13:55       831      
android-3.3.2-javadoc.jar.asc.md5                 2024-11-07 13:55        32      
android-3.3.2-javadoc.jar.asc.sha1                2024-11-07 13:55        40      
android-3.3.2-javadoc.jar.asc.sha256              2024-11-07 13:55        64      
android-3.3.2-javadoc.jar.asc.sha512              2024-11-07 13:55       128      
android-3.3.2-javadoc.jar.md5                     2024-11-07 13:55        32      
android-3.3.2-javadoc.jar.sha1                    2024-11-07 13:55        40      
android-3.3.2-javadoc.jar.sha256                  2024-11-07 13:55        64      
android-3.3.2-javadoc.jar.sha512                  2024-11-07 13:55       128      
android-3.3.2.pom                                 2024-11-07 13:55      1183      
android-3.3.2.pom.asc                             2024-11-07 13:55       831      
android-3.3.2.pom.asc.md5                         2024-11-07 13:55        32      
android-3.3.2.pom.asc.sha1                        2024-11-07 13:55        40      
android-3.3.2.pom.asc.sha256                      2024-11-07 13:55        64      
android-3.3.2.pom.asc.sha512                      2024-11-07 13:55       128      
android-3.3.2.pom.md5                             2024-11-07 13:55        32      
android-3.3.2.pom.sha1                            2024-11-07 13:55        40      
android-3.3.2.pom.sha256                          2024-11-07 13:55        64      
android-3.3.2.pom.sha512                          2024-11-07 13:55       128  
``` 

This is blocking any builds specifying `implementation 'com.bugfender.sdk:android:3.+'`

**Comment by crodriguezworky** [comment]
Same here, ¡please help!

**Comment by IvanVakhatov** [comment]
Same

**Comment by android-imdad** [comment]
for now you can change line number 41 in android build.gradle
 
from - 
 implementation 'com.bugfender.sdk:android:3.+'
 
To - 
 implementation 'com.bugfender.sdk:android:3.3.3'

**Comment by jgimenez** [comment]
Hi there,
As you mentioned, the maven package for version 3.3.2 released yesterday was malformed and was missing the .aar file.
Our engineers released version 3.3.3 with the corrected package a few minutes ago.
Please try rebuilding your project, and that should fix the problems you are experiencing. If you keep experiencing any issues, please let me know so we can look into it.
